src/Init/System/IO.lean

```diff
@@ -462,6 +462,17 @@ Note that it is the caller's job to remove the file after use.
 -/
 @[extern "lean_io_create_tempfile"] opaque createTempFile : IO (Handle × FilePath)
 
+/--
+Creates a temporary directory in the most secure manner possible. There are no race conditions in the
+file’s creation. The file is readable and writable only by the creating user ID. Additionally
```

Comment:
While directories are file's on UNIX we should probably refer to directory here? Also if the directory was not executable by anyone that would be rather bad. After all you need executable permissions to work with the directory.

Reply:
Thank you - I believe the new version fixes this.